drivers/hyperv/powershell.go

```diff
@@ -38,12 +38,12 @@ func cmdOut(args ...string) (string, error) {
 	return stdout.String(), err
 }
 
-func cmd(args ...string) error {
+func Cmd(args ...string) error {
```

Comment:
This seems like an internal function. It's very specifically only for running powershell, even if the name doesn't mention that at all.